### PR TITLE
add BIP 341 Taproot

### PR DESCRIPTION
**Pull request by MarnixCroes**

### CODE REVIEW

**Comment by yahiheb** [comment]
This is waiting for https://github.com/zkSNACKs/WalletWasabi/pull/9070 I guess.

**Comment by yahiheb** [comment]
What's up with this one @MarnixCroes ?

**Comment by MarnixCroes** [comment]
> What's up with this one @MarnixCroes ?

let's merge it afer Taproot is available for the user in the GUI

**Comment by MarnixCroes** [comment]
let's merge this
> let's merge it afer Taproot is available for the user in the GUI

I've created this issue for this

File: docs/using-wasabi/BIPs.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ Here is a list of all the supported and integrated Bitcoin Improvement Proposals
 - [BIP 158: Block filters](https://github.com/bitcoin/bips/blob/master/bip-0158.mediawiki)
 - [BIP 173: Base32 address format for native v0-16 witness outputs](https://github.com/bitcoin/bips/blob/master/bip-0173.mediawiki)
 - [BIP 174: Partially Signed Bitcoin Transaction Format](https://github.com/bitcoin/bips/blob/master/bip-0174.mediawiki)
+- [BIP 341: Taproot](https://github.com/bitcoin/bips/blob/master/bip-0341.mediawiki)
```
Comment (yahiheb) [inline docs/using-wasabi/BIPs.md:34]:
```suggestion
- [BIP 341: Taproot: SegWit version 1 spending rules](https://github.com/bitcoin/bips/blob/master/bip-0341.mediawiki)
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

**Review by yahiheb — APPROVED** [review]

**Pull request merged**